src/helpers.rs

```diff
@@ -928,6 +928,30 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
         interp_ok(())
     }
 
+    /// Check the number of fixed args.
```

Comment:
```suggestion
    /// Check the number of fixed args of a vararg function.
```
This should probably error if the function is *not* a vararg fn?

Please also put "vararg" in the name then, e.g. `check_vargarg_fixed_arg_count`.

(We used `arg_count` before so please don't use `args_count` here now.)